backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -259,12 +352,87 @@ const parseTemplateMessageWebhook = async (
   }
 }
 
+const sendPasscodeMessage = async (
```

Comment:
Can KIV for now but I think in the future we would need to perform a refactor to distinguish between whatsapp and sms functions?

Reply:
Agree to KIV.